internal/provider/resource_gitlab_group_access_token.go

```diff
@@ -236,6 +227,6 @@ func resourceGitlabGroupAccessTokenDelete(ctx context.Context, d *schema.Resourc
 	}
 
 	log.Printf("[DEBUG] Delete gitlab GroupAccessToken %s", d.Id())
-	_, err = client.GroupAccessTokens.DeleteGroupAccessToken(group, groupAccessTokenId, gitlab.WithContext(ctx))
+	_, err = client.GroupAccessTokens.RevokeGroupAccessToken(group, groupAccessTokenId, gitlab.WithContext(ctx))
```

Comment:
What's the reason for changing the behavior here? When we changed something kike that in other resources we considered jt a breaking change and added a flag to control the destroy behavior ... Wdyt?

Reply:
This is just an upstream name change in go-gitlab. No functionality has changed as far as I can tell.

Reply:
Timo this was you! You renamed it! 😆

Reply:
O.o I thought @svanharmelen did that as a tweak to my PR for Personal Access Tokens.
https://github.com/xanzy/go-gitlab/commit/4c72f522ec4f6a37769d8fbb6e942f2e5afd15fd

Reply:
It was me, see the other comment... Sorry for the inconvenience!

Reply:
Oh gosh, sorry for my slander 😭

Reply:
Hahah, was just going crazy over here about this 😅 thanks @svanharmelen for the rescue 👋